packages/composites/por-indexer/README.md

```diff
@@ -0,0 +1,66 @@
+# Bitcoin Proof-of-Reserves Indexer Service Adapter
```

Comment:
We should remove Bitcoin here, or change the name of the adapter to reference Bitcoin.

Reply:
Oh yes, will remove. We'll be using the same adapter for Dogecoin.